modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -193,7 +193,11 @@ def soc_opt_in
 
     # 15. YOU MUST INDICATE BELOW EACH ISSUE...
     def contestable_issues
-      form_data&.dig('included')
+      issues = form_data.dig('included') || []
+
+      issues.map do |issue|
```

Comment:
Let's memoize the built ContestableIssues array, so that we don't have to recreate the instances each call.